torch/testing/_internal/common_methods_invocations.py

```diff
@@ -857,7 +857,35 @@ def sample_inputs_diff(op_info, device, dtype, requires_grad, **kwargs):
 
     return tuple(sample_inputs)
 
-def sample_inputs_index_select(op_info, device, dtype, requires_grad, **kwargs):
+def sample_inputs_gradient(op_info, device, dtype, requires_grad):
+    test_cases = (
+        ((S,), None, None),
```

Comment:
Sample inputs will need to be revised based on implementation revisions. How long do the grad and gradgrad checks take for this operation?

Reply:
done